### PR TITLE
Highlight the current menu item in dropdown menu on mouse over

### PR DESCRIPTION
**Pull request by rajnikantsolarwinds**

### CODE REVIEW

**Comment by grossmj** [comment]
The contextual menu automatically opens when hovering a node, this is not fixing what the user has reported.

You can see this clearly by switching to the light theme:

Screenshot from 2022-08-18 22-08-08

and then try to select something in the contextual menu, the hovered menu item should be highlighted (it seems to work in the dark theme but could be better)

Screenshot from 2022-08-18 22-08-27

File: src/app/cartography/widgets/node.ts [hunk 1]
```diff
@@ -103,6 +103,8 @@ export class NodeWidget implements Widget {
       })
       .on('mouseover', function (this, n: MapNode) {
         select(this).attr('class', 'over');
+        // self.onContextMenu.emit(new NodeContextMenu(event, n));
```
Comment (grossmj) [inline src/app/cartography/widgets/node.ts:106]: You can remove these I think

File: src/app/components/project-map/project-map.component.scss [hunk 2]
```diff
@@ -382,6 +382,9 @@ g.node text,
 .context-menu-items .mat-menu-item:focus {
   background: none;
 }
+.context-menu-items .mat-menu-item:hover {
```
Comment (grossmj) [inline src/app/components/project-map/project-map.component.scss:385]: Does this apply only to the light theme?

**Review by grossmj — CHANGES_REQUESTED** [review]

File: src/app/components/project-map/project-map.component.scss (hunk 2 above)
Comment (rajnikantsolarwinds) [reply src/app/components/project-map/project-map.component.scss:385]: I will try @grossmj but we are using same menu with same class in light and dark both theme.

**Pull request merged**